pysb/bng.py

```diff
@@ -137,8 +137,18 @@ def _format_action_args(cls, **kwargs):
             Arguments and values to supply to BNG
         """
         if kwargs:
-            action_args = ','.join('%s=>%s' % (k, BngConsole._bng_param(v))
-                                   for k, v in kwargs.items())
+            processed_pars = []
+            for k, v in kwargs.items():
+                bng_param = BngConsole._bng_param(v)
+                if isinstance(bng_param, dict):
```

Comment:
Ideally, this should be `if isinstance(bng_param, collections.Mapping)` since it supports e.g. `OrderedDict`

Reply:
Sounds good. Thanks for the feedback